src/Tribe/Aggregator/Processes/Batch_Imports.php

```diff
@@ -51,6 +51,10 @@ public function allow_batch_import( $service_supports_batch_push, $abstract ) {
 			return $service_supports_batch_push;
 		}
 
+		if ( 'async' === tribe_get_option( 'tribe_aggregator_import_process_system' ) ) {
```

Comment:
Is this the correct approach? I do not think disabling batch-imports when the user selects async import system should be the way to go; I suggest we force the use of the correct import system (non-async, then) when the user sets up a batch import.
The improvement we're trying to implement here should not be stopped by another feature that we have all the elements to control in the code.
Am I reading this wrong?

Reply:
Great question.

Based on the document that described Batch pushing, Batch pushing relies on Cron Imports, meaning that:

1. A user has Cron import as Import system
1.1 Creates an import
1.1 This import is going to be a batch import system
1. The user changes his import system to Async mode
1.1 Now the import is processed using the `async` system instead.

Basically depending on the type of system on the installation it would select which type of Queue is going to be used to process the imports. 

Let me know if you have additional questions.

Reply:
Per hangout on 2020-11-26 : review with Strategy what the intended user flow and experience should be.

Reply:
Based on strategy: We are releasing this as part only of Cron imports and make it widely available for all type of imports on a the next release. 

Let me know in case you have additional questions, thanks. :)